src/scss/attendee/variables.scss

```diff
@@ -1,5 +1,7 @@
 $sparkle-theme: rgb(99, 42, 199);
 
+$font-size-default: 14px;
```

Comment:
We should probably start organizing these variables soon, as we make PRs. Something like in the old `constants.scss` but maybe even in separate files like "fonts.scss", "borders.scss" and put `--`.

More about this kind of discussion at https://doesitsparkle.slack.com/archives/C031U4S7D08/p1645028664349089

Reply:
++ let's do it